python/tvm/relay/ir_pass.py

```diff
@@ -245,21 +247,17 @@ def create_function_pass(pass_name, opt_level, pass_func):
     if not isinstance(pass_func, (types.FunctionType, types.LambdaType)):
         raise TypeError("pass_func must be a callable for Module pass")
 
-    return _ir_pass.CreateFunctionPass(pass_name, opt_level, pass_func)
+    return _ir_pass.CreateFunctionPass(pass_info, pass_func)
 
 
-def create_sequential_pass(pass_name, opt_level, sequential_passes,
-                           disabled=None):
+def create_sequential_pass(pass_info, sequential_passes, disabled=None):
     """Create a sequential pass using a defined optimization function from
     Python.
 
     Parameters
     ----------
-    pass_name : str
-        The name of the pass.
-
-    opt_level : int
-        The optimization level of this pass.
+    pass_info : PassInfo
```

Comment:
I think we still use a flattened params in here, because pass info is an internal object that user do not need to think of

Reply:
done.